content/en/docs/setup/install/operator/index.md

```diff
@@ -75,25 +73,214 @@ accessible to `istioctl` by using this command:
 $ istioctl experimental profile list
 {{< /text >}}
 
-Optionally, you can use the `-s` flag with a install package path to see the
-list of configuration profiles available for other Istio versions:
+## Customize Istio settings using the IstioControlPlane API
+
+You can change a feature or component setting by using the [IstioControlPlane API](/docs/reference/config/istio.operator.v1alpha12.pb/)
+([proto](https://github.com/istio/operator/blob/release-1.3/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto)).
```

Comment:
It's intentional - we want this to render as 
_IstioControlPlane_(_proto_)
where both of these are links.

Reply:
@geeknoid will nested links work?

Reply:
It's not nested, it's two links one after the other so I think it'll render.

However, I would prefer that we remove the proto link. We don't expect customers to be looking at protos, they should be looking at the reference docs on istio.io

Reply:
Is there a way to prevent the reordering of the proto? The alphabetical sort is of limited value imo and it makes the proto hard to absorb top down i.e. the most important stuff should be at the top.

Reply:
@geeknoid I guess I was thrown off by the `(` and the `)` in parans.